frontend/src/__mocks__/mockCatalogModel.ts

```diff
@@ -1,368 +0,0 @@
-import { ArtifactsProtocol, CatalogModel } from '~/concepts/modelCatalog/types';
-
```

Comment:
I think we need this `mockCatalogModel.ts` mock for cypress tests. As we are removing the `mockData` folder as part of [RHOAIENG-18959](https://issues.redhat.com/browse/RHOAIENG-18959)